react/tsconfig.json

```diff
@@ -2,7 +2,7 @@
 	"compilerOptions": {
 		/* Visit https://aka.ms/tsconfig to read more about this file */
 
-		"jsx": "react",
+		"jsx": "react-jsx",
```

Comment:
This was to get the components to compile the same as in the main UI, but can be removed if need be